classes/OccurrenceIndividual.php

```diff
@@ -1190,14 +1272,14 @@ public function restoreRecord(){
 					$sql1 = 'INSERT INTO ommaterialsample(';
 					$sql2 = 'VALUES(';
 					foreach($secArr as $f => $v){
-						if(in_array(strtolower($f),$msFieldArr)){
-							$sql1 .= $f.',';
-							$sql2 .= '"'.$this->cleanInStr($v).'",';
+						if(in_array(strtolower($f), $msFieldArr)){
```

Comment:
I have a feeling strtolower is one of those methods that's gonna give us a deprecation warning if we don't null check $f.